core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -218,6 +220,24 @@ public MethodMetrics(ResourceMethod method, PerformanceMetric annotation, Metric
           "The maximum request latency in ms", metricTags);
       this.requestLatencySensor.add(metricName, new Max());
 
+      Percentiles percs = new Percentiles(4 * 100,
```

Comment:
nit, using Float.SIZE / 8 instead of 4 and defining a constant numBuckets = 100 would be more explicit about how this relates to the underlying storage bytes.

Reply:
sure